test/operators/update-fee.ts

```diff
@@ -0,0 +1,183 @@
+    await trackGas(ssvNetworkContract.cancelDeclaredOperatorFee(1), [GasGroup.REGISTER_OPERATOR]);
+  });
+
+  it('Cancel declared fee without a pending request reverts "NoPendingFeeChangeRequest"', async () => {
+    await expect(ssvNetworkContract.cancelDeclaredOperatorFee(1
+    )).to.be.revertedWith('NoPendingFeeChangeRequest');
+  });
+
+  it('Cancel declared fee of operator I do not own reverts "CallerNotOwner"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).cancelDeclaredOperatorFee(1
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  //Execute 
+  it('Execute declared fee emits "OperatorFeeExecution"', async () => {
+    await ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).executeOperatorFee(1
+    )).to.emit(ssvNetworkContract, 'OperatorFeeExecution');
+  });
+
+  it('Execute declared fee gas limits', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod);
+    await trackGas(ssvNetworkContract.executeOperatorFee(1), [GasGroup.REGISTER_OPERATOR]);
+  });
+
+  it('Execute declared fee of operator I do not own reverts "CallerNotOwner"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).executeOperatorFee(1
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  it('Execute declared fee without a pending request reverts "NoPendingFeeChangeRequest"', async () => {
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('NoPendingFeeChangeRequest');
+  });
+
+  it('Execute declared fee too early reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod - 10);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  it('Execute declared fee too late reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod + helpers.CONFIG.executeOperatorFeePeriod + 1);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  //Dao
```

Comment:
Please make a jira ticket on yourself that we want to split this file into 2 one for operator and one for DAO
Once we do that we will remove "DAO" from all these titles

Reply:
https://bloxxx.atlassian.net/browse/IO1-2057 (: